src/DogStatsd.php

```diff
@@ -65,8 +65,8 @@ public function __construct(array $config = array())
         $this->host = isset($config['host']) ? $config['host'] : 'localhost';
         $this->port = isset($config['port']) ? $config['port'] : 8125;
         $this->datadogHost = isset($config['datadog_host']) ? $config['datadog_host'] : 'https://app.datadoghq.com';
-        $this->apiCurlSslVerifyHost = isset($config['curl_ssl_verify_host']) ? $config['curl_ssl_verify_host'] : 2;
-        $this->apiCurlSslVerifyPeer = isset($config['curl_ssl_verify_peer']) ? $config['curl_ssl_verify_peer'] : 1;
+        $this->curlVerifySslHost = isset($config['curl_ssl_verify_host']) ? $config['curl_ssl_verify_host'] : 2;
+        $this->curlVerifySslPeer = isset($config['curl_ssl_verify_peer']) ? $config['curl_ssl_verify_peer'] : 1;
```

Comment:
It was probably intended to use these fields instead:

https://github.com/DataDog/php-datadogstatsd/blob/0c4060a5eb8a141a8fc33b354f624cde06cf52a8/src/DogStatsd.php#L41-L48

That is, without the change proposed in this PR, the following statement only clears values:

https://github.com/DataDog/php-datadogstatsd/blob/0c4060a5eb8a141a8fc33b354f624cde06cf52a8/src/DogStatsd.php#L342-L343